src/_includes/soft_mod_husbpot_in_page_banner--blog_cta.liquid

```diff
@@ -0,0 +1,19 @@
+{%- if site.lang == 'es' -%}
+  <span class="hs-cta-wrapper hubspot-in-page-banner" id="hs-cta-wrapper-77a4144a-4c2a-44ce-8468-d9341d623e46">
+    <span class="hs-cta-node hs-cta-77a4144a-4c2a-44ce-8468-d9341d623e46" id="hs-cta-77a4144a-4c2a-44ce-8468-d9341d623e46">
+      <!--[if lte IE 8]><div id="hs-cta-ie-element"></div><![endif]-->
+      <a href="https://cta-redirect.hubspot.com/cta/redirect/3042464/77a4144a-4c2a-44ce-8468-d9341d623e46" ></a>
+    </span>
+    <script charset="utf-8" src="https://js.hscta.net/cta/current.js"></script>
+    <script type="text/javascript"> hbspt.cta.load(3042464, '77a4144a-4c2a-44ce-8468-d9341d623e46', {}); </script>
+  </span>
+{%- else -%}
+  <span class="hs-cta-wrapper hubspot-in-page-banner" id="hs-cta-wrapper-ad686d9b-1d11-4ef5-a68e-1cf1f0aff255">
+    <span class="hs-cta-node hs-cta-ad686d9b-1d11-4ef5-a68e-1cf1f0aff255" id="hs-cta-ad686d9b-1d11-4ef5-a68e-1cf1f0aff255">
+      <!--[if lte IE 8]><div id="hs-cta-ie-element"></div><![endif]-->
+      <a href="https://cta-redirect.hubspot.com/cta/redirect/3042464/ad686d9b-1d11-4ef5-a68e-1cf1f0aff255" ></a>
+    </span>
+    <script charset="utf-8" src="https://js.hscta.net/cta/current.js"></script>
+    <script type="text/javascript"> hbspt.cta.load(3042464, 'ad686d9b-1d11-4ef5-a68e-1cf1f0aff255', {}); </script>
+  </span>
+{%- endif -%}
```

Comment:
Following on from the previous comment, now we have repetition in 6 places. Let's have a single include that takes ID and classname?

Reply:
👍